src/rsz/src/Rebuffer.cc

```diff
@@ -1786,170 +1739,174 @@ std::vector<Instance*> Rebuffer::collectImportedTreeBufferInstances(
 
 // Martin (2024-04-18): This is copied over from original RepairSetup
 // buffering mostly unchanged and is ripe for clean-up/rewrite later.
+// Jaehyun (2025-11-27): Re-implemented using insertBufferBeforeLoads to
+// simplify hierarchy handling and avoid direct use of hierarchicalConnect.
```

Comment:
@jhkim-pii I think you have now done the rewrite this code was waiting for so we can drop the comments